client/tests/webdriver/specs/userLogin.spec.js

```diff
@@ -0,0 +1,20 @@
+import Home from '../pages/home.page'
+import { expect } from 'chai'
+
+describe('Anet user login', () => {
+	it('Default user Erin is logged in"', () => {
```

Comment:
I would leave out the user's name here and in the next test (since the actual user is defined somewhere else, and could be changed so the name here is no longer accurate).

Reply:
Done.